.prospector.yaml

```diff
@@ -0,0 +1,19 @@
+pep257:
+  disable:
+    - D203
+    - D212
+    - D213
+    - D214
+    - D215
+    - D404
+    - D405
+    - D406
+    - D407
+    - D408
+    - D409
+    - D410
+    - D411
+    - D413
+    - D415
+    - D416
+    - D417
```

Comment:
Is there any reference to why these numbers?

Reply:
[A codacy moderator was recommending these settings](https://community.codacy.com/t/d203-and-d213-are-not-required-by-pep-257/237/7) because some of them actually conflict with each other, for example:

```
    D203: 1 blank line required before class docstring

    D211: No blank lines allowed before class docstring
```
This list of settings is supposedly compatible with PEP257.